client/lib/Utility.dart

```diff
@@ -1,5 +1,26 @@
 // File for utility functions that might be usefull anywhere in the code.abstract
+import 'dart:async';
 import 'dart:math';
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+import 'package:plantexpert/api/ApiConnection.dart';
+import 'package:plantexpert/api/ApiConnectionException.dart';
+import 'package:plantexpert/api/Plant.dart';
+import 'package:plantexpert/api/UserPlant.dart';
+
+class PlantenApi {
```

Comment:
Please remove this class. We already have an `ApiConnection` class containing all the necessary functions.